frontend/src/components/Shared/DashboardSideNavbar/DashboardSideNavbar.jsx

```
@@ -0,0 +1,229 @@
+  const renderToggleButton = () => {
+    if (isMobile) return null;
+    
+    const buttonClass = `p-1 rounded-lg bg-gray-800 border-gray-700 text-white hover:bg-gray-700 ${
+      !isOpen && !isHovering ? 'mx-auto' : ''
+    }`;
+    
+    return (
+      <button
+        onClick={() => setIsOpen(!isOpen)}
+        className={buttonClass}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)